docs/specification/draft/client/elicitation.mdx

```diff
@@ -334,14 +334,14 @@ URL mode elicitation enables servers to direct users to external URLs for out-of
   elicitation URL the server wants them to open.
 </Note>
 
-URL mode elicitation requests **MUST** specify `mode: "url"` and include these additional parameters:
+URL mode elicitation requests **MUST** specify `mode: "url"`, a `message`, and include these additional parameters:
```

Comment:
No substantive change here, just clarifying that `message` (described earlier in the document` is still a required field for this type of `elicitation/create` request.